src/glass/src/assets/i18n/de_DE.po

```diff
@@ -0,0 +1,240 @@
+# 
```

Comment:
I assume the files are auto generated - can we generate them during build rather than commit every change in these files with git?

Reply:
Maybe something to improve in a follow up PR. This PR already contains things that were planned for milestone 3.